Makefile

```diff
@@ -0,0 +1,180 @@
+	@./dev/registry.sh update dev 	
+
+find-missing-license:
+	@./build/find-missing-license.sh
+
+find-missing-version:
+	@./build/find-missing-version.sh
+	
+
+###############
+# CI Commands #
+###############
+build-spark-base:
+	@docker build . -f images/spark-base/Dockerfile -t cortexlabs/spark-base:latest
+	
+build-tf-base:
+	@docker build . -f images/tf-base/Dockerfile -t cortexlabs/tf-base:latest
+	@docker build . -f images/tf-base-gpu/Dockerfile -t cortexlabs/tf-base-gpu:latest
+
+build-base: spark-base tf-base
+
+build-tf-dev:
+	@./build/build-image.sh images/tf-train tf-train
+	@./build/build-image.sh images/tf-serve tf-serve
+	@./build/build-image.sh images/tf-api tf-api
+	@./build/build-image.sh images/tf-train-gpu tf-train-gpu
+	@./build/build-image.sh images/tf-serve-gpu tf-serve-gpu
+
+build-spark-dev:
+	@./build/build-image.sh images/spark spark
+	@./build/build-image.sh images/spark-operator spark-operator
+
+build-tf-images: build-tf-base build-tf-dev
+build-spark-images: build-spark-base build-spark-dev
+
+build-argo-images:
+	@./build/build-image.sh images/argo-controller argo-controller
+	@./build/build-image.sh images/argo-executor argo-executor
+
+build-operator-images:
+	@./build/build-image.sh images/operator operator
+	@./build/build-image.sh images/nginx-controller nginx-controller
+	@./build/build-image.sh images/nginx-backend nginx-backend
+	@./build/build-image.sh images/fluentd fluentd
+
+build-images: build-tf-images build-spark-images build-argo-images build-operator-images
+
+push-tf-dev:
+	@./build/push-image.sh images/tf-train tf-train
+	@./build/push-image.sh images/tf-serve tf-serve
+	@./build/push-image.sh images/tf-api tf-api
+	@./build/push-image.sh images/tf-train-gpu tf-train-gpu
+	@./build/push-image.sh images/tf-serve-gpu tf-serve-gpu
+
+push-spark-dev:
+	@./build/push-image.sh images/spark spark
+	@./build/push-image.sh images/spark-operator spark-operator
+
+push-tf-images: push-tf-base push-tf-dev
+push-spark-images: push-spark-base push-spark-dev
+
+push-argo-images:
+	@./build/push-image.sh images/argo-controller argo-controller
+	@./build/push-image.sh images/argo-executor argo-executor
+
+push-operator-images:
+	@./build/push-image.sh images/operator operator
+	@./build/push-image.sh images/nginx-controller nginx-controller
+	@./build/push-image.sh images/nginx-backend nginx-backend
+	@./build/push-image.sh images/fluentd fluentd
+
+push-images: push-tf-images push-spark-images push-argo-images push-operator-images
+
+build-and-upload-cli:
+	@./build/cli.sh
+
+test-go:
+	@./build/test.sh go
+
+test-python:
+	@./build/test.sh python
+
+test-license:
+	@if [ "$$(./build/find-missing-license.sh)" ]; then echo "some files are missing license headers, run 'make find-missing-license' to find offending files."; exit 1; fi
+
+test-version:
+	@if [ "$$(./build/find-missing-version.sh)" ]; then echo "there are still CORTEX_VERSION references to master. run 'make find-missing-version'"; exit 1; fi
```

Comment:
Is it possible to have `find-missing-version.sh` run the check and if it fails, do the echo and `exit 1`?